<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playscene</class>
 <widget class="QMainWindow" name="Playscene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <widget class="QLabel" name="label_finish">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1700</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1700</width>
      <height>1000</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1700</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/over.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pause">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1700</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1700</width>
      <height>1000</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1700</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/pic/intro2.bmp);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toolButton_ppause">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>-10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/pause.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="toolButton_restart">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>520</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/ppause.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_speed">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>660</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/big.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/rec/pic/big.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>620</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/big.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/big.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/small.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/pic/diamond.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/pic/diamond.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>680</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/rec/pic/diamond.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/time.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>690</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/bag.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>720</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/bag.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/title-time.png);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_sum">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/title-money.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_final">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>440</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 63 72pt &quot;Adobe 繁黑體 Std B&quot;;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toolButton_cancel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>660</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/quit2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="toolButton_reset">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>800</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/rec/pic/restart.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>toolButton_ppause</zorder>
   <zorder>label_13</zorder>
   <zorder>label_3</zorder>
   <zorder>label_15</zorder>
   <zorder>label_14</zorder>
   <zorder>label_12</zorder>
   <zorder>label_11</zorder>
   <zorder>label_10</zorder>
   <zorder>label_7</zorder>
   <zorder>label_6</zorder>
   <zorder>label_8</zorder>
   <zorder>label_1</zorder>
   <zorder>label_2</zorder>
   <zorder>toolButton</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>label_pause</zorder>
   <zorder>toolButton_restart</zorder>
   <zorder>lcdNumber_speed</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>lcdNumber_sum</zorder>
   <zorder>label_finish</zorder>
   <zorder>label_final</zorder>
   <zorder>toolButton_reset</zorder>
   <zorder>toolButton_cancel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
